.github/workflows/validate.yml

```diff
@@ -66,7 +66,7 @@ jobs:
   build_on_rhel:
     strategy:
       matrix:
-        compiler: [gnu12, gnu13]
+        compiler: [gnu12]
```

Comment:
Don't disable gnu13. We want it for the future and I am aware that it currently does not work with everything.